HousingRepairsOnlineApi/Helpers/NotificationConfiguration/BaseNotificationConfigurationProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.UseCases;
+
+namespace HousingRepairsOnlineApi.Helpers.SendNotifications;
+
+public class BaseNotificationConfigurationProvider
+{
+    public string ConfirmationSmsTemplateId { get; set; }
+    public string ConfirmationEmailTemplateId { get; set; }
+    public string InternalEmailTemplateId { get; set; }
+
+    protected BaseNotificationConfigurationProvider(string confirmationSmsTemplateId, string confirmationEmailTemplateId,
+        string internalEmailTemplateId)
+    {
+        ConfirmationSmsTemplateId = confirmationSmsTemplateId;
+        ConfirmationEmailTemplateId = confirmationEmailTemplateId;
+        InternalEmailTemplateId = internalEmailTemplateId;
+    }
+
+    protected async Task<string> GetImageLink(Repair repair, IRetrieveImageLinkUseCase retrieveImageLinkUseCase)
```

Comment:
This is not related to configuration and I don't think it belongs here - it doesn't use any of the 
It's more suited to an extension method on `Repair`.